### PR TITLE
added  base classes  in messenger-impl module

### PR DESCRIPTION
**Pull request by AlexKazmirchuk**

### CODE REVIEW

File: softgroup/services/messenger/messenger-impl/src/main/java/com/softgroup/messenger/impl/handler/DeleteConversationRequestHandler.java [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+ * @since 26.02.17.
+ */
+
+public class DeleteConversationRequestHandler
+        extends AbstractRequestHandler<DeleteConversationRequest,DeleteConversationResponse>
+        implements MessengerRequestHandler {
+
+    @Override
+    public String getName() {
+        return "create_conversation";
```
Comment (Dominys) [inline softgroup/services/messenger/messenger-impl/src/main/java/com/softgroup/messenger/impl/handler/DeleteConversationRequestHandler.java]: @AlexKazmirchuk delete ?

File: softgroup/services/messenger/messenger-impl/src/main/java/com/softgroup/messenger/impl/handler/GetConversationDetailsRequestHandler.java [hunk 2]
```diff
@@ -0,0 +1,33 @@
...
+ * @since 26.02.17.
+ */
+
+public class GetConversationDetailsRequestHandler
+        extends AbstractRequestHandler<GetConversationDetailsRequest,GetConversationDetailsResponse>
+        implements MessengerRequestHandler {
+
+    @Override
+    public String getName() {
+        return "create_conversation";
```
Comment (Dominys) [inline softgroup/services/messenger/messenger-impl/src/main/java/com/softgroup/messenger/impl/handler/GetConversationDetailsRequestHandler.java]: @AlexKazmirchuk get ?

File: softgroup/services/messenger/messenger-impl/src/main/java/com/softgroup/messenger/impl/handler/GetConversationSettingsRequestHandler.java [hunk 3]
```diff
@@ -0,0 +1,32 @@
...
+ * @since 26.02.17.
+ */
+
+public class GetConversationSettingsRequestHandler
+        extends AbstractRequestHandler<GetConversationSettingsRequest,GetConversationSettingsResponse>
+        implements MessengerRequestHandler {
+
+    @Override
+    public String getName() {
+        return "create_conversation";
```
Comment (Dominys) [inline softgroup/services/messenger/messenger-impl/src/main/java/com/softgroup/messenger/impl/handler/GetConversationSettingsRequestHandler.java]: @AlexKazmirchuk ?

File: softgroup/services/messenger/messenger-impl/src/main/java/com/softgroup/messenger/impl/handler/GetConversationsByIdsRequestHandler.java [hunk 4]
```diff
@@ -0,0 +1,32 @@
...
+ * @since 26.02.17.
+ */
+
+public class GetConversationsByIdsRequestHandler
+        extends AbstractRequestHandler<GetConversationsByIdsRequest,GetConversationsByIdsResponse>
+        implements MessengerRequestHandler {
+
+    @Override
+    public String getName() {
+        return "create_conversation";
```
Comment (Dominys) [inline softgroup/services/messenger/messenger-impl/src/main/java/com/softgroup/messenger/impl/handler/GetConversationsByIdsRequestHandler.java]: @AlexKazmirchuk ?

File: softgroup/services/messenger/messenger-impl/src/main/java/com/softgroup/messenger/impl/handler/GetConversationsDetailsRequestHandler.java [hunk 5]
```diff
@@ -0,0 +1,32 @@
...
+ * @since 26.02.17.
+ */
+
+public class GetConversationsDetailsRequestHandler
+        extends AbstractRequestHandler<GetConversationsDetailsRequest,GetConversationsDetailsResponse>
+        implements MessengerRequestHandler {
+
+    @Override
+    public String getName() {
+        return "create_conversation";
```
Comment (Dominys) [inline softgroup/services/messenger/messenger-impl/src/main/java/com/softgroup/messenger/impl/handler/GetConversationsDetailsRequestHandler.java]: @AlexKazmirchuk ?

File: softgroup/services/messenger/messenger-impl/src/main/java/com/softgroup/messenger/impl/handler/GetConversationsRequestHandler.java [hunk 6]
```diff
@@ -0,0 +1,32 @@
...
+ * @since 26.02.17.
+ */
+
+public class GetConversationsRequestHandler
+        extends AbstractRequestHandler<GetConversationsRequest,GetConversationsResponse>
+        implements MessengerRequestHandler {
+
+    @Override
+    public String getName() {
+        return "create_conversation";
```
Comment (Dominys) [inline softgroup/services/messenger/messenger-impl/src/main/java/com/softgroup/messenger/impl/handler/GetConversationsRequestHandler.java]: @AlexKazmirchuk ?

File: softgroup/services/messenger/messenger-impl/src/main/java/com/softgroup/messenger/impl/handler/GetConversationsSettingsRequestHandler.java [hunk 7]
```diff
@@ -0,0 +1,33 @@
...
+ * @since 26.02.17.
+ */
+
+public class GetConversationsSettingsRequestHandler
+        extends AbstractRequestHandler<GetConversationsSettingsRequest,GetConversationsSettingsResponse>
+        implements MessengerRequestHandler {
+
+    @Override
+    public String getName() {
+        return "create_conversation";
```
Comment (Dominys) [inline softgroup/services/messenger/messenger-impl/src/main/java/com/softgroup/messenger/impl/handler/GetConversationsSettingsRequestHandler.java]: @AlexKazmirchuk ?

File: softgroup/services/messenger/messenger-impl/src/main/java/com/softgroup/messenger/impl/handler/GetMessagesRequestHandler.java [hunk 8]
```diff
@@ -0,0 +1,32 @@
...
+ * @since 26.02.17.
+ */
+
+public class GetMessagesRequestHandler
+        extends AbstractRequestHandler<GetMessagesRequest,GetMessagesResponse>
+        implements MessengerRequestHandler {
+
+    @Override
+    public String getName() {
+        return "create_conversation";
```
Comment (Dominys) [inline softgroup/services/messenger/messenger-impl/src/main/java/com/softgroup/messenger/impl/handler/GetMessagesRequestHandler.java]: @AlexKazmirchuk ?

File: softgroup/services/messenger/messenger-impl/src/main/java/com/softgroup/messenger/impl/handler/IsTypingInChatRequestHandler.java [hunk 9]
```diff
@@ -0,0 +1,33 @@
...
+ * @since 26.02.17.
+ */
+
+public class IsTypingInChatRequestHandler
+        extends AbstractRequestHandler<IsTypingInChatRequest,IsTypingInChatResponse>
+        implements MessengerRequestHandler {
+
+    @Override
+    public String getName() {
+        return "create_conversation";
```
Comment (Dominys) [inline softgroup/services/messenger/messenger-impl/src/main/java/com/softgroup/messenger/impl/handler/IsTypingInChatRequestHandler.java]: @AlexKazmirchuk ?

File: softgroup/services/messenger/messenger-impl/src/main/java/com/softgroup/messenger/impl/handler/MessagesReadConfirmationRequestHandler.java [hunk 10]
```diff
@@ -0,0 +1,33 @@
...
+ * @since 26.02.17.
+ */
+
+public class MessagesReadConfirmationRequestHandler
+        extends AbstractRequestHandler<MessagesReadConfirmationRequest,MessagesReadConfirmationResponse>
+        implements MessengerRequestHandler {
+
+    @Override
+    public String getName() {
+        return "create_conversation";
```
Comment (Dominys) [inline softgroup/services/messenger/messenger-impl/src/main/java/com/softgroup/messenger/impl/handler/MessagesReadConfirmationRequestHandler.java]: @AlexKazmirchuk ?

File: softgroup/services/messenger/messenger-impl/src/main/java/com/softgroup/messenger/impl/handler/SendMessageRequestHandler.java [hunk 11]
```diff
@@ -0,0 +1,33 @@
...
+ * @since 26.02.17.
+ */
+
+public class SendMessageRequestHandler
+        extends AbstractRequestHandler<SendMessageRequest,SendMessageResponse>
+        implements MessengerRequestHandler {
+
+    @Override
+    public String getName() {
+        return "create_conversation";
```
Comment (Dominys) [inline softgroup/services/messenger/messenger-impl/src/main/java/com/softgroup/messenger/impl/handler/SendMessageRequestHandler.java]: @AlexKazmirchuk ?

File: softgroup/services/messenger/messenger-impl/src/main/java/com/softgroup/messenger/impl/handler/UpdateConversationSettingsRequestHandler.java [hunk 12]
```diff
@@ -0,0 +1,33 @@
...
+ * @since 26.02.17.
+ */
+
+public class UpdateConversationSettingsRequestHandler
+        extends AbstractRequestHandler<UpdateConversationSettingsRequest,UpdateConversationSettingsResponse>
+        implements MessengerRequestHandler {
+
+    @Override
+    public String getName() {
+        return "create_conversation";
```
Comment (Dominys) [inline softgroup/services/messenger/messenger-impl/src/main/java/com/softgroup/messenger/impl/handler/UpdateConversationSettingsRequestHandler.java]: @AlexKazmirchuk ?

Comment (AlexKazmirchuk) [reply softgroup/services/messenger/messenger-impl/src/main/java/com/softgroup/messenger/impl/handler/UpdateConversationSettingsRequestHandler.java]: @Dominys fixed all

**Pull request merged**